airbyte-integrations/bases/destination-test-lib/src/main/java/io/airbyte/integrations/base/TestDestination.java

```diff
@@ -33,11 +33,19 @@
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.commons.lang.Exceptions;
 import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.config.JobGetSpecConfig;
```

Comment:
i cleaned this one up a big while i was in the neighborhood. just finished some todos.

Reply:
also renamed this whole package to make it distinct from the source tests.